packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss

```diff
@@ -35,7 +35,9 @@
 
 	.calendar-day {
 		&.today {
-			background-color: $concrete;
+			background-color: tint($lochmara, 90%);
+			color: $lochmara;
+			border: solid 1px $lochmara;
```

Comment:
Remove those css please. The tertiary button is defined in the bootstrap theme, no need to add style

Reply:
Hi, this is no for tertiary style.
it is for Today.

Reply:
![today](https://user-images.githubusercontent.com/2328653/65130573-49072080-da30-11e9-9bad-1e507fc4232a.png)